app/db/models/user.js

```diff
@@ -228,13 +243,16 @@ module.exports = (Sequelize, db) => {
       firstName: this.getDataValue('firstName'),
       lastName: this.getDataValue('lastName'),
       profilePicture: this.getDataValue('profilePicture'),
+      graduationYear: this.getDataValue('graduationYear'),
+      major: this.getDataValue('major'),
```

Comment:
Can you manually delete these? These lines are already in `master` and idk what'd happen if they get merged in but I'd rather not find out.